components/service/experiments/src/main/java/mozilla/components/service/experiments/debug/ExperimentsDebugActivity.kt

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.experiments.debug
+
+import android.app.Activity
+import android.os.Bundle
+import android.support.annotation.VisibleForTesting
+import mozilla.components.service.experiments.Experiments
+import mozilla.components.support.base.log.logger.Logger
+
+class ExperimentsDebugActivity : Activity() {
+    private val logger = Logger(this.javaClass.canonicalName)
```

Comment:
This doesn't quite match the logger setup elsewhere in this library.
Is this a better practice to have? If so let's have a follow-up bug on standardizing on this. If not let's use the `TAG` approach we use elsewhere.